api-spec.md

```diff
@@ -105,15 +105,22 @@ See the [OpenAPI specification document](openapi/STAC.yaml).
 provided search predicates, probably containing search metadata from the `search` extension |
 
 The `/` endpoint should function as a complete `Catalog` representation of all the data contained in the API and linked 
-to in some way from root through `Collections` and `Items`.
+to in some way from root to `Collections` and `Items` through `children` and `child` Links.
 
 The `/search` endpoint is similar to the `/collections/{collectionId}/items` endpoint in OGC API - Features in that it 
 accepts parameters for filtering, however it performs the filtering across all collections. The parameters accepted are 
 the same as the Filter Parameters above, however the *[extensions](extensions/README.md)* also provide advanced querying 
 parameters.
 
-If the `/search` endpoint is implemented, it is **required** to add a link with the `rel` type set to `search` to the 
-`links` array in `/` that refers to the search endpoint in the `href` property.
+It is **required** to add to the root endpoint (`/`) object a Link in the `links` array with the `rel` type set to `search`  that refers to the search endpoint in the `href` property, with a `type` of `application/geo+json`. This Link should look like:
```

Comment:
Why was the constraint: "If the `/search` endpoint is implemented," removed? We don't implement search for example and would not be STAC API compliant anymore.

Reply:
So you aren't required to implement `/search` in order to be STAC API compliant?  What is the minimum you're required to implement to be considered STAC API compliant?

Reply:
I removed this because I though it was supposed to be "when" rather than "if"

Reply:
I don't know.

OAFeat requires /, /conformance, /collections and /collections/:id, I think.

Anyway, this change is basically a breaking change and is not part of the CHANGELOG, but I'd prefer to also call a OAFeat API returning STAC Items/Collections a STAC API although it doesn't implement search.

Reply:
Okay, re-added that language. Also changed it so that adding the root link to search is recommended instead of required -- we should have a larger discussion around what exactly the minimum requirement should be

Reply:
Why have you changed it to a recommendation?

Reply:
I'm fine with that. I forgot the "required" was there already -- I thought I'd added that.  I didn't include it because I thought that it was odd to say that implementing the /search endpoint is optional, but adding it to Links is required.